### PR TITLE
chore: deprecate glide in go targets

### PR DESCRIPTION
**Pull request by dudymas**
## what

* deprecate glide
* replace glide targets with go get/mod

## why

* supporting both go mod and glide is tricky

### CODE REVIEW

File: modules/go/Makefile.build [hunk 1]
```diff
@@ -18,13 +18,21 @@ else
...
+	$(call assert-set,GO)
+	$(GO) get
+	$(GO) mod tidy
+
+## DEPRECATED Install dependencies
+go/glide-deps: $(GLIDE)
+	@echo "DEPRECATED: Please update to 'go mod' and use go/deps"
 	$(call assert-set,GLIDE)
 	$(GLIDE) install --strip-vendor
 	$(GLIDE) update  --strip-vendor
```
Comment (osterman) [inline modules/go/Makefile.build]:
```suggestion
	exit 1
```

Comment (osterman) [reply modules/go/Makefile.build]: Maybe move to `Makefile.deprecated`

File: modules/go/Makefile.build [hunk 2]
```diff
@@ -40,9 +48,14 @@ go/deps-dev: $(GO)
...
+	
+## Clean compiled binary and dependencies
 go/clean-all: go/clean
+	$(GO) mod download
+
+## DEPRECATED Clean compiled binary and dependency
+go/glide-clean-all: go/clean
+	@echo "DEPRECATED: Please update to 'go mod' and use go/clean-all"
 	rm -rf vendor
 	rm -rf glide.lock
```
Comment (osterman) [inline modules/go/Makefile.build]:
```suggestion
	@exit 1
```

File: modules/go/Makefile.build (hunk 1 above)
Comment (dudymas) [reply modules/go/Makefile.build]: Let's just remove it altogether in that case

File: modules/go/Makefile.build [hunk 3]
```diff
@@ -18,18 +18,10 @@ else
...
-	$(GLIDE) update  --strip-vendor
-
-## Install dependencies for build
-go/deps-build:
-	$(call assert-set,GOPATH)
-	mkdir -p  $(GOPATH)/bin
-	# 2022-098-02 https://glide.sh/get no longer works (parked domain, bad SSL cert)
-	# which $(GLIDE) || (curl https://glide.sh/get | sh)
-	which $(GLIDE) || apk add --no-cache glide
+go/deps: $(GO)
```
Comment (zdmytriv) [inline modules/go/Makefile.build:20]: I see `GLIDE` at the very top of this file

**Review by aknysh — APPROVED** [review]

**Pull request merged**